src/libs/actions/IOU.ts

```diff
@@ -6096,6 +6097,14 @@ function trackExpense(params: CreateTrackExpenseParams) {
         value: recentServerValidatedWaypoints,
     });
 
+    if (isMapDistanceRequest(transaction) || isManualDistanceRequestTransactionUtils(transaction)) {
+        onyxData?.optimisticData?.push({
+            onyxMethod: Onyx.METHOD.SET,
+            key: ONYXKEYS.NVP_LAST_DISTANCE_EXPENSE_TYPE,
+            value: transaction?.iouRequestType,
```

Comment:
NAB: We only need to set this value when IOU request type is manual since map is the default.

Reply:
Not really important though, and maybe this is more clear